app/backend/server/api/v1/dates_router.py

```diff
@@ -26,6 +23,35 @@
                                               end_date=end,
                                               users_id=users_id)
 
+@router.post("/", response_model=schemas.RangeDatesResponse)
+async def post_dates_status(
+    body: schemas.PostDateDataBody,
```

Comment:
y ...
expand it here also
its ugly like that
i want to see here right away what params to look for

Reply:
whtat?! this is not how this works :o